src/main/java/bits/bits/team/Main.java

```diff
@@ -70,6 +70,8 @@ public void onEnable() {
     if (dataSettings.isEventDiscord())
       getServer().getPluginManager().registerEvents(new EventDiscord(discord), this);
 
+    getServer().getPluginManager().registerEvents(new EventChat(), this);
```

Comment:
Idk how you setup the settings to disable the event, I just left that out for now xD.